docs/source/installing_docker.rst

```diff
@@ -79,6 +79,28 @@ You need to create the database schema by executing::
         docker exec -it kiwi_web /Kiwi/manage.py createsuperuser
 
 
+.. _configure-kiwi-base-url:
+
+Configuration of Kiwi's domain
```

Comment:
Name is `Kiwi TCMS`, was a typo in original

Reply:
OK, can I safely change line 82 like:
.. _configure-kiwi-domain: